source/patterns/@aws-solutions-constructs/aws-openapigateway-lambda/lib/index.ts

```diff
@@ -115,21 +115,21 @@ export interface OpenApiGatewayToLambdaProps {
    */
   readonly logGroupProps?: logs.LogGroupProps
   /**
-   * Optional timeout for the backing lambda function that does the OpenAPI Definition transformation.
+   * Optional user-defined timeout for the backing lambda function that does the OpenAPI Definition transformation.
    *
-   * Defaults to 3 seconds, but for larger files (tens/hundreds of megabytes or gigabytes in size) this value will need to be increased.
+   * Defaults to 1 minute, but for larger files (hundreds of megabytes or gigabytes in size) this value may need to be increased.
    *
-   * @default Duration.seconds(3)
+   * @default Duration.minutes(1)
    */
-  readonly transformTimeout?: cdk.Duration;
+  readonly internalTransformTimeout?: cdk.Duration;
   /**
-   * Optional memory size for the backing lambda function that does the OpenAPI Definition transformation.
+   * Optional user-defined memory size for the backing lambda function that does the OpenAPI Definition transformation.
```

Comment:
Lambda function custom resource installed to do the OpenAPI definition transformation. This setting does not affect the deployed architecture - only the ability for the Construct to complete its work.

Reply:
update in latest commit.